src/styles/clock.scss

```diff
@@ -0,0 +1,142 @@
+.clock {
+  position: absolute;
+  height: 200px;
+  width: 200px;
+  background-color: #fff;
+  border-radius: 100%;
+  border: 10px solid #34626c;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: auto;
+
+  &__top {
+    position: absolute;
+    width: 3px;
+    height: 8px;
+    background-color: #262626;
+    left: 0;
+    right: 0;
+    margin: 0 auto;
+  }
+
+  &__right {
+    position: absolute;
+    width: 8px;
+    height: 3px;
+    background-color: #262626;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    margin: auto;
+  }
+
+  &__bottom {
+    position: absolute;
+    width: 3px;
+    height: 8px;
+    background-color: #262626;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    margin: 0 auto;
+  }
+
+  &__left {
+    position: absolute;
+    width: 8px;
+    height: 3px;
```

Comment:
Don't duplicate yourself, Almost all the styles are the same for all elements. 
Can't you just add the general class for all of them or create a mixin?